modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -2004,7 +2005,7 @@ private CompletableFuture<Void> handleChangePendingAssignmentEvent(
 
         Set<Assignment> stableAssignments = stableAssignmentsBytes == null
                 // This is for the case when the first rebalance occurs.
-                ? ((List<Set<Assignment>>) ByteUtils.fromBytes(tblCfg.assignments().value())).get(partId)
+                ? Assignments.fromBytes(tblCfg.assignments().value()).get(partId)
                 : ByteUtils.fromBytes(stableAssignmentsBytes);
```

Comment:
This one still uses ByteUtils, is this necessary?

Reply:
`stableAssignmentsBytes` is taken from different meta storage entries, managed by rebalance. If it is `null`, assignments for partition are taken from table configuration. Serialization was reworked only for table configuration, not for rebalance keys.